docs/faq/repositories/how-do-i-reanalyze-my-repository.md

```diff
@@ -26,7 +26,7 @@ To reanalyze a branch in your repository:
 
     ![Selecting the last commit on a branch](images/reanalyze-repository-commits-list.png)
 
-1.  Click the icon next to the **Current status** of the commit to trigger a reanalysis. Codacy will display a message when the analysis is complete.
+1.  Click the **Reanalyze** button to trigger a reanalysis.
```

Comment:
This is now a link.
Also, can we describe the expected behavior when clicking the link? 
![image](https://github.com/codacy/docs/assets/26439949/dc99ca2a-750f-4f25-8252-a851984ae5b6)

Leaving a proposal for your evaluation:

```suggestion
1.  Click the **Reanalyze** link to trigger a reanalysis. Codacy will display the status of the reanalysis request.
```